src/Serialization/PolyhedralGeometry.jl

```diff
@@ -46,6 +46,41 @@ function load_internal(s::DeserializerState, ::Type{LinearProgram{T}}, dict::Dic
     return LinearProgram{T}(fr, lp, Symbol(conv))
 end
 
+##############################################################################
+@registerSerializationType(MixedIntegerLinearProgram{fmpq})
+
+function save_internal(s::SerializerState, milp::MixedIntegerLinearProgram)
+    milp_coeffs = milp.polymake_milp.LINEAR_OBJECTIVE
+    coeffs_serialized = Polymake.call_function(Symbol("Core::Serializer"), :serialize, milp_coeffs)
+    coeffs_jsonstr = Polymake.call_function(:common, :encode_json, coeffs_serialized)
+
+    return Dict(
+        :feasible_region => save_type_dispatch(s, milp.feasible_region),
+        :convention => milp.convention,
+        :milp_coeffs => JSON.parse(coeffs_jsonstr)
+    )
```

Comment:
I think you need another address parameter, the integer variables. There can be several milps with the same coefficient vector, but different integer variables.